frontend/src/container/ListOfDashboard/DashboardsList.tsx

```diff
@@ -704,6 +705,13 @@ function DashboardsList(): JSX.Element {
 							intercomMessageDisabled
 						/>
 					</Flex>
+					{isCloudUser() && (
+						<div className="integrations-container">
```

Comment:
Do we need 2 divs here? Can be handled with one parent div as well.

Reply:
@YounixM I’m following the same structure as the integration button. The class specificity is due to the styling done in the integration style, with both sharing the same structure. The easiest and best way to avoid repetition of the style is by using two divs with class names.